docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
+TIME_START        start time of the observation stored as number of seconds after the time reference in the header  yes?
+TIME_STOP         end time of the observation in the same forma as TIME_START                                       yes?
+TIME_OBSERVATION  duration of the observation                                                                       yes?
+TIME_LIVE         duration of the observation without dead time                                                     yes?
+TRIGGER_RATE      average trigger rate of the system                                                                no
+MEAN_TEMPERATURE  average temperature of the atmosphere                                                             no
+N_TELS            number of telescopes participating in the observation                                             yes?
+TEL_LIST          string with a CSV list of telescope IDs participating in the observation                          yes?
+QUALITY           data quality; recommended: "spectral" or "detection" (not used by Gammapy at the moment)          no
+================  ================================================================================================  =========
 
-* `LibreOffice Calc <http://www.libreoffice.org/discover/calc/>`_ 
-* `TOPCAT <http://www.star.bris.ac.uk/~mbt/topcat/>`_ or `STILTS <http://www.star.bris.ac.uk/~mbt/stilts/>`_
-* `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
-* Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
+Extra user defined columns are allowed; Gammapy will ignore them.
 
-A run list must have at least a column called ``Run``::
- 
-   Run
-   42
-   43
+In order for the extra columns to have full meaning the following is needed:
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+ * Extra row right after the column name, specifying the unit of the quantity listed on each column.
+ * A header with at least the following keywords:
+
+================  ==========================================================================================================================================================================================================
+keyword           description
+================  ==========================================================================================================================================================================================================
+OBSERVATORY_NAME  name of the observatory where the observations were taken. This is important for instance for coordinate transformations between celestial (i.e. RA/dec) and terrestrial (i.e. az/alt) coordinate systems.
```

Comment:
I would suggest to add a `Required?` column here as well.

`OBSERVATORY_NAME` is optional, it's not needed for many applications.

We will probably extend this to include things like the [scheme](https://github.com/gammapy/gammapy/blob/e287ca5d7df86189ee288ccefa8e2dbdbe4ac153/gammapy/obs/datastore.py#L128) and "chain", "version", "config" (see [here](https://hess-confluence.desy.de/confluence/display/HESS/HESS+Open+Source+Tools+-+HESS+data+and+IRFs+in+FITS+format#HESSOpenSourceTools-HESSdataandIRFsinFITSformat-File/folderformat)), i.e. all the info that's needed to access the right files via the `DataStore`.

@mapazarr - I know improving data format documentation and code in `gammapy.obs` is a lot of work, but this observation and data management stuff is actually the bigger part of building background models from off runs. At the core building background models just means stacking and smoothing histograms filled from event-lists, i.e. this will not be 1000s of lines of code and it's OK and even needed for you to spend more time on the observation and data management handling.